### PR TITLE
LS-75:  S3 redirect for static.livesafemobile.com

### PR DESCRIPTION
**Pull request by mike-spainhower**
- index.html and error.html for root of static domain
- js to template into error.html for redirection
- tasks to test and deploy assets to S3
### Testing
1. Delete index.html and error.html from S3
2. run `npm run deploy`
3. See that index.html and error.html are now on S3

### CODE REVIEW

File: dist/error.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE HTML>
+<html lang="en-US">
+    <head>
+        <meta charset="UTF-8">
+        <script type="text/javascript">
+'use strict';
+
+function performRedirect(location) {
+    var newLocation = location.pathname.replace(/^\/[0-9]+/, '/1');
```
Comment (shettysuraj) [inline dist/error.html]: Can we document that the '1' is defaulting to LiveSafe

File: lib/get-s3-api.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+var _ = require('ls-lodash'),
+    aws = require('aws-sdk'),
+    s3Config = {};
```
Comment (shettysuraj) [inline lib/get-s3-api.js:5]: Where are the S3 creds?

Comment (mike-spainhower) [inline lib/get-s3-api.js:5]: http://blogs.aws.amazon.com/security/post/Tx3D6U6WSFGOK2H/A-New-and-Standardized-Way-to-Manage-Credentials-in-the-AWS-SDKs

Comment (shettysuraj) [inline lib/get-s3-api.js:5]: ah, very nice. Need to do this across the board.

Comment (mike-spainhower) [inline lib/get-s3-api.js:5]: bonus, if you install aws cli it will ask you for creds and store them for you IIRC

Comment (mike-spainhower) [inline lib/get-s3-api.js:5]: aws cli is badass, install it and run `aws s3 ls`.  yes.  awesome.

Comment (mike-spainhower) [inline lib/get-s3-api.js:5]: I use the credentials file locally and will use the ENV vars on CI

**Comment by shettysuraj** [comment]
:shipit:

**Comment by mike-spainhower** [comment]
I remember why this is WIP, I was going to write tests.  Deferring this for now, once LS-72 is merged.

**Pull request merged**